DigitalLearningSolutions.Web/Styles/shared/searchableElements/sort.scss

```diff
@@ -1,99 +1,62 @@
 @import "~nhsuk-frontend/packages/core/all";
 @import "common";
 
-.sort-dropdown {
-  border-radius: 4px;
+.sort-box-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  width: 100%;
+
+  @include mq($until: tablet) {
+    flex-direction: column;
+  }
 }
 
 .sort-by-dropdown {
   @extend .left-dropdown;
-}
-
-.sort-direction-dropdown {
-  @extend .input-with-no-js-submit-button;
-
-  width: 63%;
-
-  @include mq($until: desktop) {
-    .js-enabled & {
-      width: 70%;
-    }
-  }
 
-  @include mq($from: $xl-desktop) {
-    width: 74%;
+  @include mq($until: tablet) {
+    width: 100%;
   }
-}
 
-.sort-direction-dropdown-container {
-  @include mq($from: desktop) {
-    width: calc(59% - 12px);
-    float: left;
+  @include mq($from: tablet) {
+    flex: 1;
   }
 }
 
-.sort-submit {
-  @extend .right-submit-button;
-}
-
-.sort-row {
+.sort-order-button-container {
   display: flex;
   flex-direction: row;
+  justify-content: flex-end;
 
-  @include mq($until: desktop) {
-    margin-top: 36px;
-    margin-left: 16px;
-    margin-right: 16px;
-    flex-direction: column;
-
-    .js-enabled & {
-      flex-direction: row;
-    }
-  }
-
-  @include mq($until: 339px) {
-    flex-direction: column !important;
+  @include mq($from: tablet) {
+    width: 50%;
   }
 }
 
-.sort-select-container {
-  padding-right: 8px;
-  padding-top: 8px;
+.sort-direction-dropdown {

```

Comment:
Can this just be:
```
  @extend .input-with-no-js-submit-button;
   width: 100%;
  @include mq($from: tablet) {
    flex: 1;
  }
```

Similarly with sort-by-dropdown (no need to define until and from - usually, the default is defined and then the overrides. In mobile-first dev the defaults are mobile and then there are overrides for bigger screens)